Core/include/Acts/Detector/CylindricalContainerBuilder.hpp

```diff
@@ -54,12 +54,10 @@ class CylindricalContainerBuilder : public IDetectorComponentBuilder {
 
   /// The final implementation of the cylindrical container builder
   ///
-  /// @param roots [in,out] the detector root volumes
   /// @param gctx The geometry context for this call
   ///
   /// @return an outgoing detector component
```

Comment:
I pack the root volumes into the DetectorComponent, makes the interface clearer.